src/Driver/Pdo/AbstractCommand.php

```diff
@@ -24,7 +23,7 @@
  *
  * It also provides methods for binding parameter values and retrieving query results.
  */
-abstract class AbstractCommandPDO extends AbstractCommand implements CommandPDOInterface
+abstract class AbstractCommand extends \Yiisoft\Db\Command\AbstractCommand implements CommandInterface
```

Comment:
I'm not right with naming
Better 
prefix: Abstract
type: Pdo
class Command


AbstractPdoCommand
AbstractPdoConnection
etc...

in subpackages 
PdoCommand
PdoConnection

and from other drivers (for example odbc)
OdbcCommand
OdbcConnection

Reply:
Not doing the renaming is fine for me as well as doing it.

Reply:
@terabytesoftw, @darkdef decide it please.